Neos.Neos/Classes/Package.php

```diff
@@ -76,6 +76,7 @@ public function boot(Bootstrap $bootstrap)
         $dispatcher->connect(Node::class, 'nodeRemoved', ContentCacheFlusher::class, 'registerNodeChange');
         $dispatcher->connect(Node::class, 'beforeNodeMove', ContentCacheFlusher::class, 'registerNodeChange');
 
+        $dispatcher->connect(AssetService::class, 'assetUpdated', ContentCacheFlusher::class, 'registerAssetResourceChange');
```

Comment:
We could introduce the new slot and leave `registerAssetResourceChange` deprecated as "alias"